eng/common/pipelines/templates/steps/verify-path-length.yml

```diff
@@ -0,0 +1,59 @@
+# Template for all Python Scripts in this repository
+parameters: 
+  SourceDirectory: ''
+  BasePathLength: 49
+
+steps:
+  - task: PythonScript@0
```

Comment:
As part of these being moved into eng/common we should also pull out the inline scripts into standalone script files for easier debugging and local running.